### PR TITLE
Fix elysia adapter (https://github.com/felixmosh/bull-board/pull/769)

### PR DESCRIPTION
**Pull request by kravetsone**
fixes for https://github.com/felixmosh/bull-board/pull/769

Can u help me? Im getting this error until when i comment throw logic... But it should work fine
`You've used the BullMQ adapter with a non-BullMQ queue.`

I also add `docker-compose.redis.yml` to monorepo root for more easier redis startups

### CODE REVIEW

**Comment by felixmosh** [comment]
Can you split the pr into 2?
One with the fix the other with the style changes...
It is hard to follow what is the fix

**Comment by felixmosh** [comment]
@kravetsone sorry for the trouble...
I want to revert your PR, can you make a clean PR that combines both these PRs? (the original + the fix?)

**Comment by kravetsone** [comment]
> @kravetsone sorry for the trouble...
> I want to revert your PR, can you make a clean PR that combines both these PRs? (the original + the fix?)

Do u meen merge https://github.com/felixmosh/bull-board/pull/769 and this into new PR?

**Comment by felixmosh** [comment]
No, just make a PR (#769) with this fixes from this PR.
(Sorry for that... I needed to publish other fixes before.)
